src/main/java/pages/PokemonHomePage.java

```diff
@@ -0,0 +1,25 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class PokemonHomePage {
+
+    public WebDriver driver;
+     private By Sidebar = By.className("explore");
+
+     private By ButtonCookies = By.id("onetrust-accept-btn-handler");
```

Comment:
Be more specific with the names of the elements, the attribute cannot start with a capital letter, in this case the name should tell me what element it is, for example if the button is the one with "Pokemon Tv" the name should be pokemonTvBtn